modules/ossm-verifying-the-egress-gateway-for-http-traffic.adoc

```diff
@@ -0,0 +1,177 @@
+// This procedure is used in the following assembly:
+// * gateways/ossm-getting-traffic-into-a-mesh.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-verifying-the-egress-gateway-for-http-traffic_{context}"]
+= Verifying the egress gateway for HTTP traffic
+:context: ossm-verifying-the-egress-gateway-for-http-traffic
+
+This procedure explains how to verify that the egress gateway routes traffic to the external host.
+
+.Prerequisites
+
+* You have installed a service mesh using the {SMProductName} Operator version 3.0.
+* You have created an egress gateway using gateway injection.
+
+.Procedure
+
+. Run the following command to create a namespace called `test-namespace`:
++
+[source,terminal]
+----
+$ oc create namespace test-namespace
+----
+
+. Enable sidecar injection in the namespace:
++
+[source,terminal]
+----
+$ oc label namespace test-namespace istio-injection=enabled
+----
+
+Deploy the sleep application in the namespace:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/sleep/sleep.yaml -n test-namespace
+----
+
+. Display information about the pods in the namespace:
++
+[source,terminal]
+----
+$ oc get pods -n test-namespace
+----
+
+. Store the sleep pod in the `SOURCE_POD` environment variable:
++
+[source,terminal]
+----
+$ export SOURCE_POD=$(oc get pod -n test-namespace -l app=sleep -o jsonpath='{.items[0].metadata.name}')
+----
```

Comment:
Nitpick to stay consistent with the name used in "Get traffic into a mesh" procedure.

```suggestion
. Export a `CURL_POD` environment variable that has been initialised with the name of the curl pod:
+
[source,terminal]
----
$ export CURL_POD=$(oc get pod -n test-namespace -l app=curl -o jsonpath='{.items[0].metadata.name}')
----
```

Reply:
done